ccp/control.sls

```diff
@@ -31,6 +31,32 @@ ccp_repos_dir:
   - require:
     - user: ccp_user
 
+ccp_ssh_dir:
+  file.directory:
+  - names:
+    - {{ control.dir.base }}/.ssh/
+  - mode: 700
+  - makedirs: true
+  - user: ccp
+  - require:
+    - user: ccp_user
+
+ccp_ssh_key:
+  cmd.run:
+  - name: ssh-keygen -q -N '' -f /srv/ccp/.ssh/id_rsa
```

Comment:
Filip can you please give us example? We need to bring this by default on system level together with following pull request https://github.com/Mirantis/reclass-system-salt-model/pull/67